page/partials/product_menu.ejs

```diff
@@ -0,0 +1,4 @@
+<div id="product-menu">
```

Comment:
is this file used anywhere?

Reply:
Yes, it's the right side menu. `ul` element is automatically filled by JS, for every section that has in productContent non-empty string with menu text.